did-pkh-method-draft.md

```diff
@@ -0,0 +1,256 @@
+# did:pkh Method Specification
+
+Authors: Wayne Chang, Charles Lehner, Juan Caballero
```

Comment:
Could I be added as an author? :)

Reply:
Adding! baa58f493837bf3179bcde924ad0668e6d36a0be